packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -229,6 +240,8 @@ export default function ProjectNav({
   const isCloudManager = current?.planType === PlanType.Cloud;
   const onElementClick = useOnElementClick();
 
+  const secondTabName = displayProductName();
```

Comment:
You can make this variable outside of the component!

Reply:
Sorry duh that was the whole point